assets/styles/global.scss

```diff
@@ -288,3 +288,11 @@ hr {
 .min-width-0 {
   min-width: 0;
 }
+
+.is-desktop-hidden {
+  @apply lg:hidden #{!important};
```

Comment:
Please avoid adding more classes on global.scss. You can use `lg:!hidden` on tailwind directly

Reply:
duly noted